docker/pulumi/Dockerfile

```diff
@@ -5,7 +5,7 @@ LABEL "homepage"="https://pulumi.com"
 LABEL "maintainer"="Pulumi Team <team@pulumi.com>"
 LABEL org.opencontainers.image.description="The Pulumi CLI, in a Docker container."
 
-ENV GOLANG_VERSION 1.17.3
+ENV GOLANG_VERSION 1.17.7
 ENV GOLANG_SHA256 550f9845451c0c94be679faf116291e7807a8d78b43149f9506c1b15eb89008c
```

Comment:
What is this sha256? Is it not related to go version? (asking because they are next to each other)

Reply:
Lemme check. Weird that it didn't fail the build if it is.

Reply:
ok. it failed this time which means the checks work now.